source/_posts/rails-api-backend-for-backbone.markdown

```diff
@@ -0,0 +1,337 @@
+---
+layout: post
+title: Rails API is now merged in master
+hero_image: /blog/images/heros/post-high.jpg
+comments: true
+author:
+  name: Jorge Bejar
+  email: jorge@wyeworks.com
+  twitter_handle: jmbejar
+  github_handle:  jmbejar
+  image:  /images/team/jorge-bejar.jpg
+  description: Software Engineer at Wyeworks. Ruby on Rails developer.
+published: true
+---
+
+A few weeks ago, [an announcement was made](http://wyeworks.com/blog/2015/4/20/rails-api-is-going-to-be-included-in-rails-5/) referring to the imminent inclusion of Rails API into Rails core. Until now, Rails API has been a separated project and people have been using it through the [`rails-api` gem](https://github.com/rails-api/rails-api).
+
+Finally, after some further discussion, bug fixes and last-minute changes, the [corresponding pull request](https://github.com/rails/rails/pull/19832) was merged. Therefore, the feature will be available once Rails 5 is released.
+
+Rails API goal is to facilitate the implementation of API-only Rails projects, where only a subset of Rails features are available. For example, template generation and asset pipeline are not included in a Rails API application.
+
+For more detailed information about the Rails API project, you can take a look at [this Santiago Pastorino's article](http://wyeworks.com/blog/2012/4/20/rails-for-api-applications-rails-api-released/) about the project.
+
+## How to implement a Rails API backend for a simple Backbone application.
+
+Rails 5 is not released yet, but we can give a try by using the latest code in rails' master branch. The following is a step-by-step example of how to build a Rails API backend for a simple TODO list application implemented with Backbone.
+
+Since we want to focus on the backend implementation and its integration with the client side application, we decided to borrow the [Backbone TODO application from the TodoMVC project](https://github.com/tastejs/todomvc/tree/gh-pages/examples/backbone).
+
+### Generating the Rails API-only application
+
+Let's get started! First, we need to make sure that we have the most recent version of the Rails source code (remember that, at the time of writing, this feature was just incorporated into the master branch!). The easiest way to have a copy of this code is by cloning the Rails Github project in our computer:
+
+```
+git clone git://github.com/rails/rails.git
+```
+
+Creating a new only-API application is accomplished by adding the `--api` option when generating the project with the `rails new` command. Once Rails 5 is released, you will be able to create the Rails application by running, for example, `rails new my_api_app --api`.
```

Comment:
I'd say ...

`Creating an API only application is accomplished by running rails new --api`